Source/Math/CPUMatrixImpl.h

```diff
@@ -7332,12 +7332,12 @@ template <class ElemType>
 int CPUMatrix<ElemType>::Argmax() const
 {
     int maxArg = -1;
-    ElemType maxValue = std::numeric_limits<ElemType>::min();
+    ElemType maxValue = std::numeric_limits<ElemType>::lowest();
```

Comment:
Thank you for the fix, can you update ArgMax python unit test to cover the above case?

The unit test is in: https://github.com/Microsoft/CNTK/blob/master/bindings/python/cntk/ops/tests/reduction_test.py

Reply:
Apologies, I don't have the bandwidth to get set up with a CNTK dev environment - I could modify the unit test, but wouldn't be able to run it.

Reply:
No problem I will take care of it.